docs/source/recipes/proxy.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Creating a Proxy
+---
+
+When developing Availity applications there is no use in a UI that doesn't connect to some data source. We have developed a proxy to enable you to test against real routes from other hosts from your local source.
+
+## Getting Started
+
+Most of this process is already documented [here](https://availity.github.io/availity-workflow/tutorial/mocks/), but we will do a light explanation. In the link provided you will find: how to set up your project, learn about routes for mock/proxy data, and what your terminal should look like when the server is running. Basically everything you need to know to get at least halfway out the door you will find there.
+
+## Example
+
+The best way to learn is by examples and that is what we are going to do here.
+
+We are going to clear out the `App.js` file for tesing purposes and instead paste the below code snippet.
+
+### Adding API Code Snippet
+
+```jsx header=App.js
+import React, { useEffect } from 'react';
+import { AvProxyApi } from '@availity/api-axios';
+
+const proxyApi = new AvProxyApi({ tenant: 'availity', name: '/my/proxy' });
+
+const fetchData = async () => {
+    const response = await proxyApi.query({ sessionBust: false });
+    return response;
+};
+
+const App = () => {
+    useEffect(() => {
+        fetchData();
+    }, []);
+
+    return <div>My name is...</div>;
+};
+
+export default App;
+```
+
+The above code snippet runs a method called `fetchData` on mount that will fetch the response from `AvProxyApi`.
+
+Since we know the route is going go to `api/sdk/platform/v1/users/me` we need to add our proxy route in the `routes.json` by adding the following to our file:
+
+### Adding a new Route
+
+```json header=routes.json
+{
+    "ms/api/availity/internal/spc/slotmachine/graphql": {
+        "file": "slotmachine.json"
+    },
+    "v1/proxy/availity/my/proxy": {
+        "file": "climbingholds.json"
+    }
+}
+```
+
+All proxy routes will begin with `v1/proxy/`. You will complete the route with the `tenant` and `name` you supplied when creating a new instance of AvProxyApi. In this case `tenant` would be `availity` and the `name` would be `/my/proxy` .
+
+### Adding Response Data
+
+Now that we have the route we need to test out our `climbingholds.json` response:
+
+```json header=climbingholds.json
```

Comment:
I would update the example `App.js` to use the `data` returned from here some how such as setting the climb holds in state and rendering it to the screen